numba/tests/test_withlifting.py

```diff
@@ -284,7 +284,6 @@ def test_liftcall4(self):
         msg = ("compiler re-entrant to the same function signature")
         self.assertIn(msg, str(raises.exception))
 
-    # 3.7 fails to interpret the bytecode for this example
```

Comment:
Remove the:
```
 @unittest.skipIf(PYVERSION <= (3, 8),
                     "unsupported on py3.8 and before")
```
on the line below?

Reply:
Introduced in https://github.com/numba/numba/commit/d5eb4ad50c4fb6ef504cd732c9a7960ab03cc22a, it says `test_liftcall5` should be skipped for Python 3.8 and earlier. Since we are still supporting Python 3.8, it would seem to make sense to keep this exclusion.  Are you suggesting that it actually shouldn't have been excluded when using Python 3.8?

Reply:
Thanks for pointing this out, will need to go and look at the bytecode, the comment and `@skipIf` suggest different things.

Reply:
Based on this [force push](https://github.com/numba/numba/compare/2ea4fca71348b579a691bc4087de4d77fba0d582..d5eb4ad50c4fb6ef504cd732c9a7960ab03cc22a) it appears that it was originally thought to be 3.7 and earlier issue, but was actually a 3.8 and earlier issue. Would you like me to update the comment to say `# 3.8 fails to interpret the bytecode for this example`?

Reply:
I've manually checked this against Python 3.8. In Python 3.8 it fails in bytecode tracing/analysis as the same "state" is encountered twice. Think updating the comment would be good. Thanks.

Reply:
I've added the comment back with the updated mention of 3.8 instead of 3.7.